src/fields/validations.ts

```diff
@@ -24,25 +24,34 @@ import canUseDOM from '../utilities/canUseDOM';
 import { isValidID } from '../utilities/isValidID';
 import { getIDType } from '../utilities/getIDType';
 
-export const number: Validate<unknown, unknown, NumberField> = (value: string, { t, required, min, max }) => {
-  const parsedValue = parseFloat(value);
+export const number: Validate<unknown, unknown, NumberField> = (value: number | number[], { t, required, min, max }) => {
+  const toValidate: number[] = Array.isArray(value) ? value : [value];
+
+  // eslint-disable-next-line no-restricted-syntax
+  for (const val of toValidate) {
+    const v = parseFloat(val as unknown as string);
```

Comment:
Try to avoid single character variable names if you can help it. Even in a simple function like this, it isn't as readable.